.vscode/settings.json

```diff
@@ -22,7 +22,7 @@
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "eslint.format.enable": true,
-  "breadcrumbs.enabled": false,
+  "breadcrumbs.enabled": true,
```

Comment:
is this intentional? seem unrelated to the PR

Reply:
It is intentional, see my comment above.